carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/ownership/Ownership.java

```diff
@@ -31,7 +31,10 @@
     public enum OwnerType {
         PRIMARY,
         SECONDARY
-    };
+    }
+
+    private Ownership() {
+    }
 
     public static String getMethodOwner(ITestResult result, OwnerType type) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "getMethodOwner" is 12 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.251.139/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)